### PR TITLE
feat: add a criteria of yaw error

### PR DESCRIPTION
**Pull request by ktro2828**
## Types of PR

- [x] New Features

## Description

Add a criteria of yaw error named `YawError`.

## How to review this PR

I confirmed this feature works with the latest workflow.

## Others

This PR should be reviewed after https://github.com/tier4/driving_log_replayer/pull/425 has been merged.

### CODE REVIEW

File: driving_log_replayer/driving_log_replayer/criteria/perception.py [hunk 1]
```diff
@@ -262,6 +263,24 @@ def calculate_score(frame: PerceptionFrameResult) -> float:
...
+        errors = []
+        for result in frame.object_results:
+            err = result.heading_error
+            if err is not None:
+                _, _, yaw_err = err
+                errors.append(yaw_err)
+
+        return 0.0 if len(errors) == 0 else np.mean(errors)
+
+
```
Comment (kosuke55) [inline driving_log_replayer/driving_log_replayer/criteria/perception.py:283]:
```suggestion
    @property
    def is_greater(self) -> bool:
        return False
```

is necessary?

File: driving_log_replayer/driving_log_replayer/criteria/perception.py [hunk 2]
```diff
@@ -262,6 +263,24 @@ def calculate_score(frame: PerceptionFrameResult) -> float:
...
+    name = CriteriaMethod.YAW_ERROR
+
+    def __init__(self, level: CriteriaLevel) -> None:
+        super().__init__(level)
+
+    @staticmethod
+    def calculate_score(frame: PerceptionFrameResult) -> float:
+        errors = []
+        for result in frame.object_results:
+            err = result.heading_error
```
Comment (kosuke55) [inline driving_log_replayer/driving_log_replayer/criteria/perception.py:276]: I'm not sure but this value is abs?

Comment (ktro2828) [reply driving_log_replayer/driving_log_replayer/criteria/perception.py:278]: updated in https://github.com/tier4/driving_log_replayer/pull/450/commits/4a52a29f21a2c77b4835c71f1b6e7c7678ef3cc2

File: driving_log_replayer/driving_log_replayer/criteria/perception.py (hunk 1 above)
Comment (ktro2828) [reply driving_log_replayer/driving_log_replayer/criteria/perception.py:285]: This property will be added by https://github.com/tier4/driving_log_replayer/pull/425

**Review by kosuke55 — APPROVED** [review]
LGTM

**Review by hayato-m126 — APPROVED** [review]
LGTM

**Pull request merged**